api/operator_autopilot.go

```diff
@@ -54,6 +54,45 @@ type AutopilotConfiguration struct {
 	ModifyIndex uint64
 }
 
+func (u *AutopilotConfiguration) MarshalJSON() ([]byte, error) {
+	type Alias AutopilotConfiguration
+	return json.Marshal(&struct {
```

Comment:
Why is this needed? `time.Duration` should serialize fine?

Reply:
By default it serializes to the underlying int64 (which is in nanoseconds) instead of the "3h" or "200ms" format